systemvm/debian/opt/cloud/bin/cs/CsDhcp.py

```diff
@@ -80,7 +80,7 @@ def configure_server(self):
             # DNS search order
             if gn.get_dns() and device:
                 sline = "dhcp-option=tag:interface-%s-%s,6" % (device, idx)
-                dns_list = [x for x in gn.get_dns() if x is not None]
+                dns_list = [x for x in gn.get_dns() if not (not x)]
```

Comment:
`not (not x))` Why a double negation?

Reply:
Fixed, I was trying to fix something that required `x` not be empty or None, I check again a simple `if x` works :)